src/app.js

```diff
@@ -1 +1,37 @@
-// write code here
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const [file, destination] = process.argv.slice(2);
+
+function moveFile() {
+  try {
+    if (!file || !destination) {
+      throw new Error('Both arguments are required');
+    }
+
+    if (file === destination) {
+      console.log('Source and destination files are the same');
+
+      return;
+    }
+
+    let destinationPath = path.resolve(destination);
+    const fileName = path.basename(file);
```

Comment:
please move  these `ifs` and `consts` outside the block `try`

Reply:
It doesn't work in this way